Sources/Spezi/Spezi.docc/Spezi Ecosystem.md

```diff
@@ -0,0 +1,268 @@
+- [Local Data Storage](#local-data-storage) for efficient data persistence and retrieval.
+- [Electronic Health Integration](#electronic-health-integration) for managing FHIR data and data interoperability.
+- [Mobile Health Data](#mobile-health-data) for accessing HealthKit data.
+- [LLMs and AI Integration](#llms-and-ai-integration) for AI-driven functionalities.
+- [Spezi Questionnaire](#spezi-questionnaire) for customizable forms.
+- [Spezi Medication](#spezi-medication) for managing medication details, dosages, and more.
+- [Tasks and Reminders](#tasks-and-reminders) for task scheduling and notification management.
+- [Spezi Notifications](#spezi-notifications) for implementing user notifications and related.
+
+
+## Overview of the Core Modules
+
+The Spezi ecosystem provides a collection of interoperable modules designed to accelerate healthcare and research application development. Each module is built to work independently or in combination with others, allowing you to choose the components that best fit your needs.
+
+
+> Important: To get started with Spezi, we recommend exploring the **[Spezi Template Application](https://github.com/StanfordSpezi/SpeziTemplateApplication)** first, which demonstrates many of the following modules in action.
+
+
+### Spezi Onboarding
+- The **[SpeziOnboarding](https://github.com/StanfordSpezi/SpeziOnboarding)** provides features for a comprehensive onboarding experience with welcome screens, sequential onboarding flows, and consent management.
+
+This module is included and demonstrated in the **[Spezi Template Application](https://github.com/StanfordSpezi/SpeziTemplateApplication)**
+
+@Row{
+    @Column {
+        ![Screenshot displaying the onboarding view.](https://raw.githubusercontent.com/StanfordSpezi/SpeziOnboarding/main/Sources/SpeziOnboarding/SpeziOnboarding.docc/Resources/OnboardingView.png)
+    }
+    @Column {
+        ![Screenshot displaying the sequential onboarding view.](https://raw.githubusercontent.com/StanfordSpezi/SpeziOnboarding/main/Sources/SpeziOnboarding/SpeziOnboarding.docc/Resources/SequentialOnboardingView.png)
+    }
+    @Column {
+        ![Screenshot displaying the consent view.](https://raw.githubusercontent.com/StanfordSpezi/SpeziOnboarding/main/Sources/SpeziOnboarding/SpeziOnboarding.docc/Resources/ConsentView.png)
+    }
+}
+
+> Tip: Check out the [SpeziOnboarding Documentation](https://swiftpackageindex.com/StanfordSpezi/SpeziOnboarding/documentation/spezionboarding) to learn more about the implementation details.
+
+
+### User Management
+- The **[SpeziAccount](https://github.com/StanfordSpezi/SpeziAccount)** module provides comprehensive user authentication and profile management capabilities, enabling secure login flows, account creation, and profile customization within your application.
+
+This module is included and demonstrated in the **[Spezi Template Application](https://github.com/StanfordSpezi/SpeziTemplateApplication)**.
```

Comment:
As we use this sentence quite often, we might figure out another neat way of displaying this without repeating the phrase all the time.
Maybe some custom label "included in Template Application" next to the module heading would be great?
To be more specific, I could imagine something similar to our visual CI indicators:
<img width="485" alt="image" src="https://github.com/user-attachments/assets/8d040347-6dcb-488b-b02e-513c19dbcf2a" />